test/t8n/t8n.cpp

```diff
@@ -117,15 +119,28 @@ int main(int argc, const char* argv[])
 
                         if (loaded_tx_hash_opt != computed_tx_hash)
                             throw std::logic_error("transaction hash mismatched: computed " +
-                                                   hex0x(computed_tx_hash) + ", expected " +
+                                                   computed_tx_hash_str + ", expected " +
                                                    hex0x(loaded_tx_hash_opt.value()));
                     }
 
+                    if (tracing_enabled)
+                    {
+                        // Remove old tracers
+                        vm.set_option("stdtrace", "no");
+                        auto output_filename =
```

Comment:
Can we use some tmpfile feature so it gets deleted afterwards?

Reply:
Could use [std::tmpfile()](https://en.cppreference.com/w/cpp/io/c/tmpfile).

Reply:
This returns a file and opens it. We need only a file name here. Moreover we cannot delete it because retesteth needs to read it.